lib/jsdom/living/cssom/CSSStyleDeclaration.webidl

```diff
@@ -0,0 +1,12 @@
+// https://drafts.csswg.org/cssom/#the-cssstyledeclaration-interface
+[Exposed=Window]
+interface CSSStyleDeclaration {
+  [CEReactions] attribute CSSOMString cssText;
+  readonly attribute unsigned long length;
+  getter CSSOMString item(unsigned long index);
```

Comment:
If I am reading the diffs correctly, this means that `el.style[0]` will now work, whereas it never did before. Am I correct? If so, adding a quick test case or two to the WPTs would be nice.

Reply:
That's true in theory.
But how do we set a getter for this[i]?

Reply:
Got it fixed for Element.style (but not for CSSStyleRule.style)